pcdscalc/misc_calcs.py

```diff
@@ -0,0 +1,351 @@
+import numpy as np
+from .be_lens_calcs import alias
+
+# TODO: find a better number
+_manual_energy = 9.8
```

Comment:
This guy keeps coming back, the `_manual_energy` - for now I gave it a random number, but in the old code this is `None` -  i should probably change this back to `None`?

Reply:
... no clue ... is this a 'default photon energy' when not specified?

Reply:
Yeah it seems like this is a default photon energy when not specified. The `9.8` number is just a random number I put it mostly for testing, but it is gone now.